urdf/include/urdf/model.h

```diff
@@ -34,39 +34,42 @@
 
 /* Author: Wim Meeussen */
 
-#ifndef URDF_MODEL_H
-#define URDF_MODEL_H
+#ifndef URDF__MODEL_H_
+#define URDF__MODEL_H_
 
 #include <string>
-#include <map>
+
 #include <urdf_model/model.h>
 #include <urdf/urdfdom_compatibility.h>
 #include <tinyxml.h>
 #include <boost/shared_ptr.hpp>
 #include <boost/weak_ptr.hpp>
 #include <ros/ros.h>
 
-namespace urdf{
+namespace urdf
+{
 
-class Model: public ModelInterface
+class Model : public ModelInterface
 {
 public:
   /// \brief Load Model from TiXMLElement
-  bool initXml(TiXmlElement *xml);
+  bool initXml(TiXmlElement * xml);
```

Comment:
if being ROS2 coherent, then we should have a new line between return type and function. Don't feel strong about it though.

Reply:
As someone who does that a lot, I'd say it's optional. My rule is that I will separate the return type and the method name with a new line if:

- the class already does this for another function
- the function is too long for 100 characters, I'll do this first
- a function was changed to this style, I will change the others to match (consistent within a class or file)

If the functions/methods are all short (under 100 characters), then not wrapping is fine.

----

Also, in this particular case the code is from another existing project and therefore I'd say it doesn't _have_ to change, especially since this is something the linters will not catch.

By the same conventions (if you were to strictly follow the ROS 2 style) the function would be `init_xml` too, rather than `initXml`.

Reply:
I'd like to keep the differences with upstream to a minimum, particularly if the linters don't care about it.  So I'm going to leave it for now.